src/app/extend/context.ts

```
@@ -270,9 +270,12 @@
 
 declare module '@eggjs/core' {
   interface Context {
+    // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)